src/components/TradeWidget/TxMessage.tsx

```diff
@@ -158,6 +158,13 @@ const useLowVolumeAmount = ({ sellToken, sellTokenAmount, networkId }: LowVolume
   const gasPrice = useGasPrice({ defaultGasPrice: DEFAULT_GAS_PRICE, gasPriceLevel: 'fast' })
 
   return useMemo(() => {
+    if (priceEstimation !== null && priceEstimation.isZero()) {
+      // no price data for token
+      logDebug('No priceEstimation data for', sellToken.symbol, 'in OWL')
+
+      return { isLoading: false, isLowVolume: false }
```

Comment:
So if there's no price estimation, we don't show the warning at all?
When does this occur? It's because an issue with the service?
I mean, gasPrice we should always have, so it's basically the price of ETH in OWL. Should we have a fallback too for this?

Reply:
> happens when that token has not been traded yet

No issue with the service

with ETH it's not a problem, even in case it is reported as 0 (which won't happen), warning won't show as `minTradableVolume` will be calced to 0.